modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/HiveLoadAlphaFoldDBProteinFeatures.pm

```diff
@@ -0,0 +1,196 @@
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+=head1 NAME
+
+HiveLoadAlphaFoldDBProteinFeatures.pm
+
+=head1 DESCRIPTION
+
+This module inserts protein features into an Ensembl core database based on the AlphaFoldDB-UniProt mappings found in the EMBL-EBI AlphaFoldDB SIFTS data and the UniProt-ENSP mappings found in the GIFTS database in order to make the link between AlphaFoldDB and ENSP having a AlphaFoldDB entry as a protein feature for a given ENSP protein.
+
+It also populates the "pdb_ens" table in the GIFTS database with similar data.
+
+=head1 OPTIONS
+
+-alpha_path     Path to file containing the AlphaFoldDB mapping data (including file name).
+
+-core_dbhost    Core database host name.
+
+-core_dbport    Core database port.
+
+-core_dbname    Core database name.
+
+-core_dbuser    Core database username to connect as.
+
+-core_dbpass    Core database password to use.
+
+-cs_version     Coordinate system version.
+
+-species        Production name of species to process
+
+-rest_server    GIFTS rest server to fetch the perfect matches data from.
+
+=head1 EXAMPLE USAGE
+
+standaloneJob.pl Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveLoadAlphaFoldDBProteinFeatures -alpha_path /hps/nobackup/flicek/ensembl/mr6/AlphaFoldDB/homo_sapiens/alpha_mappings.txt -core_dbhost genebuild3 -core_dbport 4500 -core_dbname carlos_homo_sapiens_core_89_test -core_dbuser *** -core_dbpass *** -cs_version GRCh38 -species homo_sapiens -rest_server https://www.ebi.ac.uk/gifts/api/
+
+=cut
+
+package Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveLoadAlphaFoldDBProteinFeatures;
+
+use strict;
+use warnings;
+
+use 5.014002;
+use Bio::EnsEMBL::Analysis::Tools::Utilities;
+use parent ('Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveBaseRunnableDB');
+use Bio::EnsEMBL::Analysis::Runnable::MakeAlphaFoldDBProteinFeatures;
```

Comment:
Change dependency to new path

Reply:
this is no Config File , should be in dedicated source folder `AlphaFold` sub module